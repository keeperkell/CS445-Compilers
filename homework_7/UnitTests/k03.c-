main()
 begin 
     for i <- 0 to 3 do  begin 
          for j <- 0 to 3 do  begin 
              output(i);
              output(j);
              outnl();
           end 
      end 
 end 
