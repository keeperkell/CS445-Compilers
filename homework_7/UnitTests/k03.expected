Number of warnings: 0
Number of errors: 0
------------------------------------
Loading file: UnitTests/k03.tm
0 0
0 1
0 2
1 0
1 1
1 2
2 0
2 1
2 2
Bye.
